manifests/charts/gateways/istio-egress/values.yaml

```diff
@@ -281,3 +281,17 @@ global:
   #   kubernetes: cluster.local
   #   else:  default dns domain
   trustDomain: "cluster.local"
+
+meshConfig:
```

Comment:
Sorry for necro-ing something ancient and maybe I am asking a FAQ but... isn't meshConfig the global config map and it can only have 1 value and so it should be defined in istio-discovery or base and read here? it seems the chart relies on values being copied multiple times? (for instance the needed
```
  proxyMetadata:
              GOMAXPROCS: "2"
```
needs to be repeated in each values.yaml instead of set in once and runtime read from global config?
)

https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/